### ISSUE TITLE
Form Preview route URL should use SSL configuration.

### ISSUE THREAD

**Issue by kjohnson** [impress-org/givewp#5188]
## User Story

As a user, I want to avoid mixed content warnings so that I can view the form in the Wizard.

## Details

Loading the Form Preview in the Onboarding Wizard throws a mixed content warning and blocks the iframe.

## Expected Behavior

The Form Preview route URL should match the http/https configuration used by the Onboarding Wizard.

## Steps to Reproduce

1. Go to the Onboarding Wizard
1. Go to Step 4, Preview

## Visuals

![image](https://user-images.githubusercontent.com/10858303/91087130-f8c01780-e61d-11ea-836a-a57c839cf712.png)

## Acceptance Criteria

-   [ ] Form preview loads with http when the wizard uses http.
-   [ ] Form preview loads with https when the wizard uses https.

**Comment by kjohnson** [comment]
Looks like `home_url()` isn't considering `is_ssl()`.

![image](https://user-images.githubusercontent.com/10858303/91087463-784de680-e61e-11ea-868f-a93fddb1b42e.png)

**Comment by kjohnson** [comment]
A plugin like [Really Simple SSL](https://wordpress.org/plugins/really-simple-ssl/) resolves the issue, which I think I would be using in production along side a certificate.

**Comment by kjohnson** [comment]
Because the Form iframe isn't in the admin, WordPress isn't honoring `is_ssl()`, rather it is defaulting to the value stored in the `home` option.

```php
$url = get_option( 'home' );
// ...
if ( is_ssl() && ! is_admin() ) {
    $scheme = 'https';
} else {
    $scheme = parse_url( $url, PHP_URL_SCHEME );
}
```

**Comment by kjohnson** [comment]
Ok, so, the change here is that I started using WP-CLI to automate refreshing my WordPress installation, in which I wasn't setting a protocol for `---url`. Previously, I was using the browser based installer, which was pulling in the `https` from the URL.

**Comment by kevinwhoffman** [comment]
@kjohnson This doesn't seem to be as serious as we first thought. Moving this out of the release and into cool-down.

**Comment by kevinwhoffman** [comment]
Closing as this was resolved and merged in #5189.
